File: sdkwork-app-sdk-typescript/src/types/funnel-step-vo.ts
/** 漏斗步骤VO */
export interface FunnelStepVO {
  /** 创建时间 */
  createdAt?: string;
  /** 更新时间 */
  updatedAt?: string;
  /** 事件名称 */
  eventName?: string;
  /** 用户数量 */
  userCount?: number;
  /** 转化率 */
  conversionRate?: number;
  /** 步骤转化率 */
  stepConversionRate?: number;
}

File: sdkwork-app-sdk-java/src/main/java/com/sdkwork/app/model/FunnelStepVO.java
package com.sdkwork.app.model;

public class FunnelStepVO {
    private String createdAt;
    private String updatedAt;
    private String eventName;
    private Integer userCount;
    private Double conversionRate;
    private Double stepConversionRate;

    public String getCreatedAt() {
        return this.createdAt;
    }
    
    public void setCreatedAt(String createdAt) {
        this.createdAt = createdAt;
    }

    public String getUpdatedAt() {
        return this.updatedAt;
    }
    
    public void setUpdatedAt(String updatedAt) {
        this.updatedAt = updatedAt;
    }

    public String getEventName() {
        return this.eventName;
    }
    
    public void setEventName(String eventName) {
        this.eventName = eventName;
    }

    public Integer getUserCount() {
        return this.userCount;
    }
    
    public void setUserCount(Integer userCount) {
        this.userCount = userCount;
    }

    public Double getConversionRate() {
        return this.conversionRate;
    }
    
    public void setConversionRate(Double conversionRate) {
        this.conversionRate = conversionRate;
    }

    public Double getStepConversionRate() {
        return this.stepConversionRate;
    }
    
    public void setStepConversionRate(Double stepConversionRate) {
        this.stepConversionRate = stepConversionRate;
    }
}
